jetson/percep/pcl.cpp

```diff
@@ -3,19 +3,19 @@
 
 #if OBSTACLE_DETECTION
 /* --- Pass Through Filter --- */
-//Filters out all points with z values that aren't within a threshold
-//Z values are depth values in mm
+//Filters out all points on a given axis that aren't within a threshold
+//Values are depth values in mm
 //Source: https://rb.gy/kkyi80
-void PCL::PassThroughFilter() {
+void PCL::PassThroughFilter(const std::string axis, const double upperLimit) {
```

Comment:
It's not clear what values I can and should pass in for `axis`. You should at very least add this to the function comment. I think it might also be worth doing some error checking.